src/main/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/courtsentencing/CourtCaseMappingDto.kt

```diff
@@ -15,9 +15,6 @@ data class CourtCaseMappingDto(
   @Schema(description = "DPS court case id", example = "123456")
   val dpsCourtCaseId: String,
 
-  @Schema(description = "Court Appearance mappings")
-  val courtAppearances: List<CourtAppearanceMappingDto> = emptyList(),
-
   @Schema(description = "Label (a timestamp for migrated ids)")
```

Comment:
changing to an 'All' mapping dto for a hierarchical mapping  - so no longer need the child entities here